docs/dev/task_failure_recovery.md

```diff
@@ -264,4 +268,44 @@ The cluster defined restart strategy is used.
 This is helpful for streaming programs which enable checkpointing.
 By default, a fixed delay restart strategy is chosen if there is no other restart strategy defined.
 
+## Failover Strategies
+
+Flink supports different failover strategies which can be configured via the configuration parameter
+*jobmanager.execution.failover-strategy* in Flink's configuration file `flink-conf.yaml`.
+
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class="text-left" style="width: 50%">Failover Strategy</th>
+      <th class="text-left">Value for jobmanager.execution.failover-strategy</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+        <td>Restart all</td>
+        <td>full</td>
+    </tr>
+    <tr>
+        <td>Restart pipelined region</td>
+        <td>region</td>
+    </tr>
+  </tbody>
+</table>
+
+### Restart All Strategy
```

Comment:
I have the feeling that a user could confuse this with a *Restart Strategy*.
How about *Restart All Failover Strategy*? If you have other ideas, let me know.

Reply:
Yes. The naming for failover strategies are confusing with restart strategies.
I don't have any good ideas yet. Let's make it _Restart All Failover Strategy_ here to reduce possible confusion.